src/ucs/config/global_opts.c

```diff
@@ -44,6 +44,7 @@ ucs_global_opts_t ucs_global_opts = {
     .tuning_path           = "",
     .memtrack_dest         = "",
     .memtrack_limit        = UCS_MEMUNITS_INF,
+    .memlock_limit         = 500 * UCS_MBYTE,
```

Comment:
`512`?

Reply:
BTW, it seems that it could be even `0`, because it will be set from configuration anyway, wdyt?

Reply:
Good catch, thanks!
But I prefer to leave it as 512 to save the consistency between the variables that are already set to their default values at this point.

Reply:
> But I prefer to leave it as 512 to save the consistency between the variables that are already set to their default values at this point.

yes, makes sense